src/script/api/script_object.hpp

```diff
@@ -34,6 +33,26 @@ typedef bool (ScriptModeProc)();
  */
 typedef bool (ScriptAsyncModeProc)();
 
+/**
+ * Simple counted object. Use it as base of your struct/class if you want to use
+ *  basic reference counting. Your struct/class will destroy and free itself when
+ *  last reference to it is released (using Release() method). The initial reference
+ *  count (when it is created) is zero (don't forget AddRef() at least one time if
+ *  not using CCountedPtr.
```

Comment:
But `CCountedPtr` doesn't exist :)

Dunno if it also needs `@api` or not.

Reply:
Ah... yes